source/test/F0.Minesweeper.Logic.Tests/TestMinelayer.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using F0.Minesweeper.Logic.Abstractions;
+using F0.Minesweeper.Logic.Minelayer;
+
+namespace F0.Minesweeper.Logic.Tests
+{
+	public class TestMinelayer : IMinelayer
```

Comment:
More of a theoretical question: `TestXYZ` vs. `XYZForTests`
I personally prefer the `XYZForTests` variant since I'm lazy and with that way of writing intellisense will give me earlier the suggestion I want 😆

Reply:
Awesome suggestion, thanks!
I'm gonna try to rename this with a `git mv` as to not truncate the change log 😉